examples/create_letter.js

```diff
@@ -5,12 +5,12 @@
  * Run me! This example works out of the box, "batteries included".
  */
 
-var fs = require('fs');
+const fs = require('fs');
 
-var lobFactory = require('../lib/index.js');
-var Lob = new lobFactory('YOUR_API_KEY');
+const lobFactory = require('../lib/index.js');
+const Lob = new lobFactory('YOUR_API_KEY');
 
-var file = fs.readFileSync(__dirname + '/html/letter.html').toString();
+const file = fs.readFileSync(`${__dirname  }/html/letter.html`).toString();
```

Comment:
nit: for this, and other string interpolation changes, could you remove spaces between the brackets? for example this would be changed to `${__dirname}`

Reply:
Sure thing! Fixed: d8a979f.